core/lib/config/src/configs/prover_job_monitor.rs

```diff
@@ -0,0 +1,182 @@
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+
+/// Config used for running ProverJobMonitor.
+/// It handles configuration for setup of the binary (like database connections, prometheus) and configuration for jobs that are being ran.
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct ProverJobMonitorConfig {
+    /// Port for prometheus metrics connection.
+    pub prometheus_port: u16,
+    /// Maximum number of database connections per pool.
+    pub max_db_connections: u32,
```

Comment:
hmmm would be good to better understand the required number of DB connections for this component. Or Is it the more the merrier? 

From the prospective of system operator, it's be good to know the valid/recommended ranges for this param. Would the component be functional with just one connection?